infrastructure/main.tf

```diff
@@ -130,7 +146,13 @@ module "claim-store-api" {
 
     // robot notifications
     RPA_NOTIFICATIONS_SENDER = "noreply@reform.hmcts.net"
-    RPA_NOTIFICATIONS_RECIPIENT = "${data.vault_generic_secret.rpa_email.data["value"]}"
+    RPA_NOTIFICATIONS_RECIPIENT_SEALED_CLAIM = "${data.vault_generic_secret.rpa_email_sealed_claim.data["value"]}"
+    RPA_NOTIFICATIONS_RECIPIENT_MORE_TIME = "${data.vault_generic_secret.rpa_email_more_time_requested.data["value"]}"
+
+    RPA_NOTIFICATIONS_DEFENCE_RESPONSE = "${data.vault_generic_secret.rpa_email_defence_response.data["value"]}"
+    RPA_NOTIFICATIONS_DEFAULT_JUDGEMENT = "${data.vault_generic_secret.rpa_email_default_judgment.data["value"]}"
+    RPA_NOTIFICATIONS_ADDITIONAL_TIME = "${data.vault_generic_secret.rpa_email_additional_time.data["value"]}"
```

Comment:
Please remove `RPA_NOTIFICATIONS_ADDITIONAL_TIME `. There is just one RPA for more time requested event.